PatientMonitoringSystem/PatientMonitoringSystem/Controllers/BedsideSystemController.cs

```diff
@@ -63,5 +64,16 @@ public void RemoveModule(Guid moduleId)
 
 			bedsideSystemView.RemoveModule(moduleId);
 		}
+
+		private IModuleStrategy CreateStrategy(string strategyName) {
+			switch (strategyName) {
```

Comment:
@harrytpstudent Switching on a string should be fine, as long as you can guarantee that the cases and the dropdown menu will be in sync. A more complex way of doing this would be to extract the string into a class for constants, or switch on an enum and dynamically populate the dropdown menu.

Reply:
Noted.  Adding after the merge to apply fix for max/min value bug.